index.js

```diff
@@ -190,6 +221,7 @@ body1div1.appendChild(body1txt1);
 const body1txt2 = document.createElement('p');
 body1txt2.style.display = 'flex';
 body1txt2.setAttribute('class', 'body1-txt2');
+body1txt2.setAttribute('id', 'body1-txt2');
 body1txt2.textContent = 'Creative commons global summit 2015';
 body1txt2.style.color = '#ec5242';
 body1txt2.style.fontFamily = 'COCOGOOSE, sans-serif';
```

Comment:
- [ ]  Let's improve on our semantic usage on this [lines of code](https://github.com/abiy006/Capstone-1/blob/183c5519890eb799dac06f7c2f47adca6dcc218a/index.js#L221-L234) of the text as this should be a h1 tag which denotes the main heading of your page and not a paragraph.

- [ ] Hey, in the mobile version of your homepage, the header text is not showing complete. please, let's improve on this so users can know exactly what the web page is all about.

![image](https://user-images.githubusercontent.com/59397159/233787360-3b48e657-c321-4b18-bfb7-52f61ce3596c.png)